src/pychart/runner.py

```diff
@@ -25,11 +25,13 @@ def run_prompt():
     try:
         while True:
             line = input("$ : ")
+            statement = line
+            while line:
```

Comment:
I think its fine for now just not a nice user experience to press enter 2 times... An alternative might be to allow a specific keybinding (shift+enter) to insert a newline to `stdin` and `stdout`. Otherwise we can also implement something along these lines

```python
while line.endsWith('{'):
   ...
```

This way it adds a newline when needed not everytime.

Reply:
yes but what about
```
1 ==
1
;
```

There is just to many edge cases

Reply:
but shift enter thing sounds nice

Reply:
Idk I feel like that should be an error lol, or maybe we make it so it waits until `;`?

Reply:
```
if(value) {
}
```